packages/webhooks/src/commands/webhooks/add.ts

```diff
@@ -37,15 +37,15 @@ export default class WebhooksAdd extends BaseCommand {
       },
     })
 
+    CliUx.ux.action.stop()
```

Comment:
Since we are stopping and re-starting, it's probably worth adding a message as a param here like `retrieving webhook secret` or something like that since we wouldn't want it to print `done` when it isn't actually done.

Reply:
I did that initially, and it added that to the stderr output - which obvs made it fail the test! But maybe it would be more suitable to move the "done" - I'll look into that.

Reply:
Interesting... I tried it locally before recommending it and the tests passed

Reply:
I gave this another shot, and I must've added it to the other stop() before. This worked - thanks!